db/migrate/20180130165200_drop_webmks.rb

```diff
@@ -0,0 +1,7 @@
+class DropWebmks < ActiveRecord::Migration[5.0]
+  class SettingsChange < ApplicationRecord; end
```

Comment:
I think we want `ActiveRecord::Base` to be the parent class in this case since migrations shouldn't be using any models.  cc @Fryguy